.travis.yml

```diff
@@ -114,12 +119,12 @@ before_install:
   - curl -fsSkL https://raw.githubusercontent.com/openmeeg/ci-utils/master/travis/travis_tools.sh > x.sh && source ./x.sh
 
   ############################################################################
-  # Install MKL (if necessary)
+  # Install MKL if necessary
   ############################################################################
 
   - |
     if [[ "${BACKEND}" == *"Intel"* ]]; then
-      curl -fsSkL https://raw.githubusercontent.com/openmeeg/ci-utils/master/travis/install_mkl.sh > install_mkl.sh && source ./install_mkl.sh
+      curl -fsSkL https://raw.githubusercontent.com/openmeeg/ci-utils/master/travis/install-mkl.sh > install_mkl.sh && source ./install_mkl.sh
```

Comment:
is this really necessary? I just want to make sure it does not break the packaging of static binaries for mac and linux

Reply:
I did not change anything in the cmake build, so it should change nothing in the packaging and especially on the static binaries. And certainly not for mac, for which I changed nothing....

Reply:
can we revert this now just to check it was not a necessary change?

Reply:
I'm pretty sure it is not a necessary change (or at least, that the old way can be made to rework easily. I suspected changes in the paths on the intal web site). On the other hand, I'm sure it is a better way of doing things. In particular, mkl libraries are put where developpers will usually find it (ie not  in the openmeeg directory). Honestly, I do not see why you are so afraid.... The build has not been changed with this respect so if it was using static libraries, if they were not there it should have errorred... In additin, it is somewhat 

If you really want to test the old way, let's do a branch just to test that... and never integrate it. The new way is just better (at it is relies on the linux - here ubuntu - packaging system).